### PR TITLE
Update gperf type instructions

### PR DESCRIPTION
**Pull request by theresa-m**
See https://github.com/eclipse-openj9/openj9/pull/17781#issuecomment-1664172951 for more details.

### CODE REVIEW

**Comment by keithc-ca** [comment]
My preference is to remove that comment and hope that my proposed fix to gperf is accepted (see https://savannah.gnu.org/bugs/index.php?64512).

**Comment by theresa-m** [comment]
Nice! Works for me. Updated the change.

**Comment by keithc-ca** [comment]
My original proposal was not accepted. Let's wait a day or two to see if my alternate proposal gets further.

File: runtime/bcutil/attrlookup.gperf [hunk 1]
```diff
@@ -24,9 +24,6 @@
  * This file is the input to the gperf utility for generating a perfect
  * hash table of attribute names, which is used by attributeTagFor().
  *
- * Types generated by gperf cause compiler issues in the OpenJ9 Windows builds,
- * use the existing types.
- *
```
Comment (keithc-ca) [inline runtime/bcutil/attrlookup.gperf:29]: The header file `attrlookup.h` should be regenerated to reflect this change.

Comment (theresa-m) [reply runtime/bcutil/attrlookup.gperf:29]: Updated.

**Comment by keithc-ca** [comment]
Jenkins test sanity win jdk17

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**